passes/techmap/abc9.cc

```diff
@@ -281,18 +267,24 @@ void abc9_module(RTLIL::Design *design, RTLIL::Module *module, std::string scrip
 	abc9_script += stringf("&read %s/input.xaig; &ps; ", tempdir_name.c_str());
 
 	if (!script_file.empty()) {
-		if (script_file[0] == '+') {
-			for (size_t i = 1; i < script_file.size(); i++)
-				if (script_file[i] == '\'')
-					abc9_script += "'\\''";
-				else if (script_file[i] == ',')
-					abc9_script += " ";
-				else
-					abc9_script += script_file[i];
-		} else
+		if (check_file_exists(script_file))
```

Comment:
So with this change we can have file names starting with `+` (cool!) and unadorned scripts (also cool!)
but if the file doesn't exist it just puts the bare file name in the script. Then abc prints the message `ABC: Generic file reader requires a known file extension to open "nonexistent.script".` but that isn't actually an error, it continues until abc9 fails with ```ERROR: Assert `existing_cell' failed in passes/techmap/abc9.cc:520.``` Do you think that is explicit enough for users to figure out the problem or should we try harder to find out if the string is meant to be a filename or a script? (E.g., can you have an abc9 script that doesn't contain `&` or `;`?)

Reply:
As discussed, reverted change so that all non-files must start with `+` (but unlike the command line, scratchpad entries can contain spaces)